src/index.html

```diff
@@ -4,9 +4,447 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="card container__card" data-qa="card">
```

Comment:
You added this container__card class for the card but it doesn't do anything, please remove it
![image](https://user-images.githubusercontent.com/54936044/230998446-e420b242-b34c-489c-baf6-dc943f3e1e57.png)